dev/Pager/InteractionTests/PagerTestsPageElements.cs

```diff
@@ -0,0 +1,250 @@
+﻿using Windows.UI.Xaml.Tests.MUXControls.InteractionTests.Infra;
+using Windows.UI.Xaml.Tests.MUXControls.InteractionTests.Common;
+using System;
+using System.Numerics;
+using Common;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+using Microsoft.Windows.Apps.Test.Automation;
+using Microsoft.Windows.Apps.Test.Foundation.Controls;
+using Microsoft.Windows.Apps.Test.Foundation.Waiters;
+using Microsoft.Windows.Apps.Test.Foundation;
+using System.Collections.Generic;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Composition;
+
+namespace Windows.UI.Xaml.Tests.MUXControls.InteractionTests
+{
+    class PagerTestsPageElements
+    {
+        public UIObject GetPager()
+        {
+            return GetElement(ref Pager, "TestPager");
+        }
+        private UIObject Pager;
+
+        public UIObject GetPagerNumberBox()
+        {
+            if (!NumberBoxDisplayModeActive)
+            {
+                Log.Warning("Must Set Pager Display Mode to NumberBox mode");
+                return null;
+            }
+
+            if (PagerNumberBox == null)
+            {
+                PagerNumberBox = GetPager().Children[5];
```

Comment:
>PagerNumberBox = GetPager().Children[5]; [](start = 16, length = 40)

Is there any way that we can change this so that if the template changes and the numberbox is no longer child 5 we would detect that this isn't working anymore?

Reply:
Yes, I've updated these functions